### PR TITLE
[Merged by Bors] - chore(Algebra/GroupWithZero/Action/Opposite): Move from `GroupTheory.GroupAction.Opposite`

### PR DESCRIPTION
**Pull request by YaelDillies**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/GroupWithZero/Action/Opposite.lean [hunk 1]
```diff
@@ -63,3 +62,4 @@ open MulOpposite
 instance CancelMonoidWithZero.toFaithfulSMul_opposite [CancelMonoidWithZero α]
     [Nontrivial α] : FaithfulSMul αᵐᵒᵖ α :=
   ⟨fun h => unop_injective <| mul_left_cancel₀ one_ne_zero (h 1)⟩
+#align cancel_monoid_with_zero.to_has_faithful_opposite_scalar CancelMonoidWithZero.toFaithfulSMul_opposite
```
Comment (mattrobball) [inline Mathlib/Algebra/GroupWithZero/Action/Opposite.lean:65]: Uh oh.

**Comment by mattrobball** [comment]
Please clean up that align before merging.

bors delegate+

File: Mathlib/Algebra/GroupWithZero/Action/Opposite.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Algebra/GroupWithZero/Action/Opposite.lean]: Tried very hard to not reintroduce them... 😓

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**